packages/react-native-renderer/src/ReactNativeHostConfig.js

```diff
@@ -26,6 +26,10 @@ import ReactNativeFiberHostComponent from './ReactNativeFiberHostComponent';
 
 const {get: getViewConfigForType} = ReactNativeViewConfigRegistry;
 
+export type ReactListenerEvent = Object;
+export type ReactListenerMap = Object;
+export type ReactListener = Object;
```

Comment:
I wonder if you considered using a shared type between the reconciler and non-reconciler based renderers?

Do we expect that the type of `ReactListenerMap` and `ReactListener` will vary between renderers? I guess they would if `ReactListenerEvent` varied... maybe it would be hard to manage this with Flow generics.

I'm just a little concerned that Flow doesn't have our back here to keep all of the renderers in sync.

Reply:
I considered it, but I think that will largely become something to consider once we actually add these to those renderers. Given this is only exported (for now anyway) on `ReactDOM`, I don't see this as a blocker. In the long-term, we'd probably expose slightly different API. Like you mention, doing this with generics can become a bit of a pain, so I wanted to keep it as `Object` for now, just to make this work-load easier to get into the internal testing phase.